MapboxNavigation/RouteMapViewController.swift

```diff
@@ -24,13 +26,29 @@ class RouteMapViewController: UIViewController {
     @IBOutlet weak var statusView: StatusView!
     @IBOutlet weak var laneViewsContainerView: LanesContainerView!
     @IBOutlet weak var rerouteFeedbackTopConstraint: NSLayoutConstraint!
+    @IBOutlet weak var endOfRouteContainer: UIView!
```

Comment:
These names don't really follow Cocoa naming conventions but they're pretty isolated and internal.
Preferably, they should end with `View`, `Constraint` etc.

Reply:
Yeah, I'm really conflicted about this. There's certainly an argument to be made for homogeneity for the sake of consistency, and I'm not even [consistent](https://github.com/mapbox/mapbox-navigation-ios/pull/848/files/5500b125705d03c92f0e947c7ed53efff0bf935f#diff-292af689ce88d5b5aa06966456abc0e9R14) within my own additions in this PR. I do believe that swift's [guideline](https://swift.org/documentation/api-design-guidelines/#promote-clear-usage) of "things should be named based upon roles, not types" is desirable, however I don't believe I did a good enough job in this PR with "swift-ier nomenclature", so I'll change this to conform to the rest of the class.

This would be a good topic to bring up in the next retro. 

- [ ] refactor names to conform to existing convention

/cc @1ec5 @bsudekum @akitchen

Reply:
<- Guilty,  could always be better at naming.

Reply:
> I do believe that swift's [guideline](https://swift.org/documentation/api-design-guidelines/#promote-clear-usage) of "things should be named based upon roles, not types" is desirable

Note that this library is written for Cocoa Touch, not pure Swift, so the Swift naming conventions sometimes have to accommodate Cocoa naming conventions. The problem with just calling it a container is that “container” is already a heavily overloaded term. “Container view” trades brevity for clarity.